frontend/fingertips-frontend/components/table/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { Table } from 'govuk-react';
+import { WeatherForecast } from '@/generated-sources/api-client';
+
+interface tableProps {
+  data: WeatherForecast[];
+}
+
+export function PlainTable({ data }: Readonly<tableProps>) {
```

Comment:
I think we (now) know that the tables will be different for different chart types, so perhaps worth calling this component `LineChartTable`?